eng/Versions.props

```diff
@@ -70,6 +70,52 @@
     <MicrosoftExtensionsHttpPollyVersion>9.0.0-rtm.24477.5</MicrosoftExtensionsHttpPollyVersion>
     <MicrosoftExtensionsObjectPoolVersion>9.0.0-rtm.24477.5</MicrosoftExtensionsObjectPoolVersion>
   </PropertyGroup>
+  
+  <PropertyGroup Label="Manual">
+    <!-- Packages from dotnet/runtime -->
+    <MicrosoftBclTimeProviderLTSVersion>8.0.1</MicrosoftBclTimeProviderLTSVersion>
```

Comment:
Do we have the any of the servicing concerns as illustrated in https://github.com/dotnet/efcore/pull/34769#issuecomment-2391622052? When we service this repo, we will need to wait for the 8.0.x release to be done before we can build this repo. Will that be a problem?

cc @ericstj

Reply:
Not really because dotnet/extensions is not part of product construction. The reason why efcore is a concern, is that efcore 9 (which is part of product construction) would now depend on the build of .NET 8 servicing, meaning that a rebuild of 8 servicing would trigger at the very least a partial rebuild for .NET 9. In our case, while .NET 8 servicing rebuild would trigger a rebuild for us as well, it would only affect our layer as nothing depends on the live versions that we produce higher in the stack, and .NEt 9 wouldn't require a rebuild as extensions is not part of it.